test/test_modules.py

```diff
@@ -110,6 +110,20 @@ def test_factory_kwargs(self, device, dtype, module_info):
                             buffer.dtype, dtype,
                             f'Buffer {name} is of dtype {buffer.dtype} instead of the expected dtype {dtype}')
 
+    @modules(module_db)
+    def test_repr(self, device, dtype, module_info):
+        # Test module can be represented with repr and str without errors.
+        module_cls = module_info.module_cls
+        module_inputs = module_info.module_inputs_func(module_info, device=device, dtype=dtype,
+                                                       requires_grad=False)
+        for module_input in module_inputs:
+            args, kwargs = module_input.constructor_input.args, module_input.constructor_input.kwargs
```

Comment:
Drive-by fyi: one thing to be considerate of in the overall design of ModuleInfo testing is test time. I'm not sure how long running through all the modules and their inputs takes but it might makes sense to glom multiple tests together to avoid having to do it so often